apiary.apib

```diff
@@ -3,245 +3,212 @@ HOST: http://lore.mit.edu/
 
 # LORE
 
-Simple API on top of MIT LORE application
+A REST API on top of MIT LORE application.
```

Comment:
nit: There is no such think as a REST API, just RESTful ones (sorry for the rando comment, just happened to be passing by)